shared/app.js

```diff
@@ -70,6 +73,8 @@ module.exports = Backbone.Model.extend({
      * Initialize the `ClientRouter` on the client-side.
      */
     if (!isServer) {
+      var ClientRouter = this.options.ClientRouter || require(defaultRouterModule);
```

Comment:
Before it had "hardcoded" default router path, so `r.js` was able to augment it and with this change it makes it invisible for `r.js`.